src/components/layouts/pages/inspect/validationErrorMessages_nl.json

```diff
@@ -118,12 +126,20 @@
         "DESCRIPTION": "Er staan lege koppen in je document. Vermijd koppen zonder tekst, dit brengt voorleessoftware in de war. Gebruik nooit lege koppen om witregels in te voegen. Witregels maak je door in het menu 'Alinea' naar de tab 'Inspringen en afstand' te gaan en daar de Afstand 'voor' of 'na' groter te maken."
       },
       "20": {
-        "SUMMARY": "Is dit een bijschrift van de figuur erboven?",
-        "DESCRIPTION": "Deze tekst lijkt bedoeld te zijn als bijschrift van de figuur erboven. Zorg dat je bijschriften als onderdeel van de figuur opmaakt. In Word en LibreOffice doe je dat door met de rechtermuisknop op de figuur te klikken en dan in het menu 'Bijschrift invoegen' te kiezen."
+        "SUMMARY": "Is dit een bijschrift van de figuur?",
```

Comment:
maybe better add "or tabel"

Reply:
Thanks for rasising this. I have changed the text accordingly.